jdi-light/src/main/java/com/epam/jdi/light/elements/base/JDIBase.java

```diff
@@ -473,16 +479,9 @@ public void actionsWithElement(JFunc1<Actions, Actions> action) {
     }
     private Safe<Actions> actions = new Safe<>(() -> new Actions(driver()));
 
-    private ElementArea clickAreaType;
-    public ElementArea getClickType() {
-        return clickAreaType != null ? clickAreaType : CLICK_TYPE;
-    }
-    public void setClickArea(ElementArea area) { clickAreaType = area; }
-    private TextTypes textType;
-    public TextTypes getTextType() {
-        return textType != null ? textType : TEXT_TYPE;
-    }
-    public void setTextType(TextTypes type) { textType = type; }
+    public ElementArea clickAreaType = CLICK_TYPE;
+    public TextTypes textType = TEXT_TYPE;
+    public SetTextTypes setTextType = SET_TEXT_TYPE;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4522228)